README.md

```diff
@@ -28,15 +34,15 @@ Written with native Asyncio NSQ package
 - [x] Sampling
 - [ ] AUTH
 
-## Usages
+## Usage
 
 ### Consumer
 
 A simple consumer reads messages from "example_topic" and prints them to stdout.
 
 ```python
-import ansq
 import asyncio
+import ansq
```

Comment:
This is minor. Usually it rather alphabetically

```python
import ansq
import asyncio
```

or two sections: one for stdlib and another for other lib sources

```python
import asyncio

import ansq
```

Reply:
Makes sense. Addresssed in 785d526